src/ontology/fbbt-edit.obo

```diff
@@ -44203,8 +44203,10 @@ relationship: part_of FBbt:00047863 ! radial vein
 id: FBbt:00004351
 name: campaniform sensillum of ventral radius Sc5
 def: "Campaniform sensillum of the medial radius on the ventral surface of the adult wing. There are five circular sensilla with low profile and with a socket (type 3), arranged linearly along the medial radius. Each sensillum is around 4.5 micrometers in diameter." [FlyBase:FBrf0027793, FlyBase:FBrf0038304]
+comment: Dinges et al. (2020) suggest that v.Rad.C be used to refer only to the 4 most proximal campaniform sensilla (campaniform sensillum of ventral radius Sc4) instead of all 5 sensilla of the ventral radius.
 subset: cur
 synonym: "sensillum campaniformium of ventral radius Sc5" EXACT []
+synonym: "v.Rad.C" NARROW [FlyBase:FBrf0247616]
```

Comment:
Reference for this synonym was originally FBrf0064796 - probably best to add back (as they actually use it for this meaning). Fine to have the Dinges ref too for why it is narrow and not exact (ok to have >1 ref per synonym).